homeassistant/components/google_assistant/helpers.py

```diff
@@ -678,10 +678,18 @@ def sync_serialize(self, agent_user_id, instance_uuid):
         elif area_entry and area_entry.name:
             device["roomHint"] = area_entry.name
 
-        # Add deviceInfo
         if not device_entry:
             return device
 
+        # Add Matter info
+        if "matter" in self.hass.config.components and (
+            matter_info := matter.get_matter_device_info(self.hass, device_entry.id)
```

Comment:
What happens here if two entities are exposed from same device?

Reply:
They will both be tagged with the same Matter unique ID. I don't know how Google would handle that.